test_search_indexer.py

```
@@ -0,0 +1,101 @@
+import os
+import sys
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+# Размер тестового батча
+TEST_BATCH_SIZE = 20
+os.environ["SEARCH_INDEXER_BATCH_SIZE"] = str(TEST_BATCH_SIZE)
+
+from app.workers.search_indexer.worker import SearchIndexerWorker  # noqa: E402
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)